app/containers/Forms/FuelFields.tsx

```diff
@@ -117,6 +141,11 @@ const FuelFields: React.FunctionComponent<Props> = ({
               </option>
             ))}
           </Form.Control>
+          <Form.Control.Feedback type="invalid">
+            <li style={{fontSize: '16px', color: '#DC3545'}}>
+              {errorSchema?.Methodology?.__errors[0]}
+            </li>
+          </Form.Control.Feedback>
         </Form.Group>
```

Comment:
Add a card to refactor these to use `FieldTemplate` and `StringField`

Reply:
https://youtrack.button.is/issue/GGIRCS-1161